### PR TITLE
image recognition

### PR DESCRIPTION
**Pull request by thulavall**
**WHY?** 
Aida image recognition functionality is to  be part of the final product.

**WHAT?**
AIDA image recognition funcitonality...

### CODE REVIEW

File: AIDA/ros2_humble_ws/src/image_recognition/image_recognition/video_analyzer.py
Comment (alexanderortengren) [inline AIDA/ros2_humble_ws/src/image_recognition/image_recognition/video_analyzer.py:1]: Fix the todo before the pull request is approved

File: AIDA/ros2_humble_ws/src/image_recognition/package.xml
Comment (alexanderortengren) [inline AIDA/ros2_humble_ws/src/image_recognition/package.xml:1]: Add Albin as maintainer of the node

File: AIDA/ros2_humble_ws/src/image_recognition/test/test_flake8.py
Comment (alexanderortengren) [inline AIDA/ros2_humble_ws/src/image_recognition/test/test_flake8.py:1]: Believe we have removed this test for all other packages. Should be removed.

File: AIDA/ros2_humble_ws/src/image_recognition/test/test_pep257.py
Comment (alexanderortengren) [inline AIDA/ros2_humble_ws/src/image_recognition/test/test_pep257.py:1]: Believe we have removed this test for all other packages. Should be removed.

File: AIDA/ros2_humble_ws/src/video/test/test_copyright.py
Comment (alexanderortengren) [inline AIDA/ros2_humble_ws/src/video/test/test_copyright.py:1]: Believe we have removed this test for all other packages. Should be removed.

File: AIDA/ros2_humble_ws/src/video/test/test_flake8.py
Comment (alexanderortengren) [inline AIDA/ros2_humble_ws/src/video/test/test_flake8.py:1]: Believe we have removed this test for all other packages. Should be removed.

File: AIDA/ros2_humble_ws/src/video/test/test_pep257.py
Comment (alexanderortengren) [inline AIDA/ros2_humble_ws/src/video/test/test_pep257.py:1]: Believe we have removed this test for all other packages. Should be removed.

File: README.md
Comment (alexanderortengren) [inline README.md:1]: Can't see what has changed, but should be fine

File: AIDA/ros2_humble_ws/src/video/video/web_video.py
Comment (alexanderortengren) [inline AIDA/ros2_humble_ws/src/video/video/web_video.py:1]: The docstring does not seem to correctly describe the node. 
Functions needs docstrings as well

File: AIDA/ros2_humble_ws/src/video/video/video_viewer.py
Comment (alexanderortengren) [inline AIDA/ros2_humble_ws/src/video/video/video_viewer.py:1]: Test node? If so create a folder called test_nodes to make it easier to distinguish. If not write docstrings.

File: AIDA/ros2_humble_ws/src/video/video/camera_video.py
Comment (alexanderortengren) [inline AIDA/ros2_humble_ws/src/video/video/camera_video.py:1]: Write docstrings for files.

File: AIDA/ros2_humble_ws/src/image_recognition/image_recognition/video_analyzer.py
Comment (alexanderortengren) [reply AIDA/ros2_humble_ws/src/image_recognition/image_recognition/video_analyzer.py:1]: Also add docstrings for functions to make it easier for next project group to understand what the class and functions are doing

File: AIDA/ros2_humble_ws/src/image_recognition/image_recognition/gesture_recognizer.py
Comment (alexanderortengren) [reply AIDA/ros2_humble_ws/src/image_recognition/image_recognition/gesture_recognizer.py:1]: You set all the variables in the init function of the class then call setup which in turn just calls the setup_camera_feed. Can we remove one step here?

File: AIDA/ros2_humble_ws/src/image_recognition/image_recognition/pose_landmarker.py
Comment (alexanderortengren) [inline AIDA/ros2_humble_ws/src/image_recognition/image_recognition/pose_landmarker.py:1]: Mostly the same comments as in gesture_recognizer. Add docstrings for node and functions. 
Is the model path hardcoded or does it work on another systems? The setup functions seems to be deprecated as you just call the setup_camera_feed with variables set in init.

**Review by alexanderortengren — CHANGES_REQUESTED** [review]
See comments on files. We also have no tests at all, not even trivial tests.

File: AIDA/ros2_humble_ws/src/image_recognition/image_recognition/video_analyzer.py
Comment (thulavall) [reply AIDA/ros2_humble_ws/src/image_recognition/image_recognition/video_analyzer.py:1]: done

File: AIDA/ros2_humble_ws/src/image_recognition/package.xml
Comment (thulavall) [reply AIDA/ros2_humble_ws/src/image_recognition/package.xml:1]: done

File: AIDA/ros2_humble_ws/src/image_recognition/test/test_copyright.py
Comment (thulavall) [reply AIDA/ros2_humble_ws/src/image_recognition/test/test_copyright.py:1]: done

File: AIDA/ros2_humble_ws/src/image_recognition/test/test_flake8.py
Comment (thulavall) [reply AIDA/ros2_humble_ws/src/image_recognition/test/test_flake8.py:1]: done

File: AIDA/ros2_humble_ws/src/image_recognition/setup.py
Comment (thulavall) [reply AIDA/ros2_humble_ws/src/image_recognition/setup.py:1]: done

File: AIDA/ros2_humble_ws/src/video/test/test_copyright.py
Comment (thulavall) [reply AIDA/ros2_humble_ws/src/video/test/test_copyright.py:1]: done

File: AIDA/ros2_humble_ws/src/video/test/test_flake8.py
Comment (thulavall) [reply AIDA/ros2_humble_ws/src/video/test/test_flake8.py:1]: done

File: AIDA/ros2_humble_ws/src/video/test/test_pep257.py
Comment (thulavall) [reply AIDA/ros2_humble_ws/src/video/test/test_pep257.py:1]: done

File: AIDA/ros2_humble_ws/src/video/video/video_viewer.py
Comment (thulavall) [reply AIDA/ros2_humble_ws/src/video/video/video_viewer.py:1]: I removed the node. It was a test node that is not needed anymore.

File: AIDA/ros2_humble_ws/src/image_recognition/image_recognition/gesture_recognizer.py
Comment (thulavall) [reply AIDA/ros2_humble_ws/src/image_recognition/image_recognition/gesture_recognizer.py:1]: done

**Review by alexanderortengren — APPROVED** [review]
Is approved. But how are we supposed to remember to remove hardcoded path for the models?

File: AIDA/ros2_humble_ws/src/video/video/video_viewer.py
Comment (alexanderortengren) [reply AIDA/ros2_humble_ws/src/video/video/video_viewer.py:1]: Maybe still have it in project for easy testing?

**Comment by thulavall** [comment]
I'll add a ticket for the removal of hardcoded paths

**Pull request merged**
